AUTHORS

```diff
@@ -226,6 +250,7 @@ Weilong Cui                             cuiwl@google.com
 Wenxian Li                              wofanli@gmail.com
 Will Deuschle                           wdeuschle@palantir.com
 Yongkun Gui                             ygui@google.com
+youssefazrak                            yazrak.tech@gmail.com
 Youssef Azrak                           yazrak.tech@gmail.com
```

Comment:
duplicate, how this should be handled?

Reply:
Probably best to add an entry to [.mailmap](../blob/master/.mailmap):

```
Youssef Azrak <yazrak.tech@gmail.com>
```